src/filters/dmx_ogg.c

```diff
@@ -699,18 +699,30 @@ GF_Err oggdmx_process(GF_Filter *filter)
 					else if (st->info.type==GF_CODECID_OPUS) {
 						block_size = gf_opus_check_frame(st->opus_parser, (char *) oggpacket.packet, oggpacket.bytes);
 						if (!block_size) continue;
+
+						if (!st->recomputed_ts) {
+							gf_filter_pid_set_property(st->opid, GF_PROP_PID_DELAY, &PROP_SINT((s32)-st->opus_parser->PreSkip));
+						}
```

Comment:
are we always sure that PreSkip is less than one block size ? If not we might need to signal preskip on several samples

Reply:
PreSkip can be more than one block size. However the edit list applies once to all the samples. What did I miss?

Reply:
once the first packet will be sent, st->recomputed_ts will no longer be null. I think you need to store the PreSkip length in the context and decrement by block_size at each packet